doozer

```diff
@@ -2187,9 +2187,12 @@ def update_operator_metadata(runtime, nvr_list, stream, merge_branch, force=Fals
 
 
 @cli.command("operator-metadata:latest-build", short_help="Print latest metadata build of a given operator")
-@click.argument("operator_list", nargs=-1, required=True)
+@click.option("--nvr", "-n", help="Specify Name-Version-Release", default=None, required=False)
```

Comment:
@sosiouxme was that your intention? When I read the ticket I also understand that it should receive a single NVR, but wouldn't be better to receive multiple ones? (maybe a RFE for the future)

Reply:
it really should be able to specify `--nvr` multiple times and look up all at once rather than have to invoke it for each, if for no other reason than for consistency, but it's not critical right now unless you need it for the jenkins work...